Subject: [PATCH] conf(.env.dev): set APP_SECRET
---
Index: .env.dev
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env.dev b/.env.dev
--- a/.env.dev	(revision 578e24eee50ae881dc0c3f72370e7edb5c7534f4)
+++ b/.env.dev	(revision cf969628933d690f4678a20f56bb2704588f81be)
@@ -1,5 +1,7 @@
 # Define your env variables for the development environment here

 ###> symfony/framework-bundle ###
-# APP_SECRET=
+# This variable is for the demo app only!
+# See https://symfony.com/doc/current/reference/configuration/framework.html#configuration-framework-secret
+APP_SECRET=not-a-secret--read-the-doc!
 ###< symfony/framework-bundle ###
