diff --git a/.env b/.env
--- a/.env	(revision c11b519660d7a269f7e33cff6296419f19ba57b0)
+++ b/.env	(revision fcf67598bfa4c7ccd280e3fcdf3bd02898ad01a4)
@@ -33,5 +33,5 @@
 # DATABASE_URL="sqlite:///%kernel.project_dir%/var/data_%kernel.environment%.db"
 # DATABASE_URL="mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=8.0.32&charset=utf8mb4"
 # DATABASE_URL="mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=10.11.2-MariaDB&charset=utf8mb4"
-DATABASE_URL="postgresql://app:!ChangeMe!@127.0.0.1:5432/app?serverVersion=16&charset=utf8"
+POSTGRES_VERSION=16
+POSTGRES_DB=app
+POSTGRES_USER=app
+POSTGRES_PASSWORD=!ChangeMe!
+POSTGRES_HOST=database
+POSTGRES_HOST_PUBLIC=127.0.0.1
+POSTGRES_PORT=5432
+POSTGRES_PORT_PUBLIC=5432
+DATABASE_URL="postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}?serverVersion=${POSTGRES_VERSION}&charset=utf8"
 ###< doctrine/doctrine-bundle ###
